data/phone/text/unknown_callee.asm

```diff
@@ -18,7 +18,7 @@ UnknownText_0x1b518b:
 	line "bit much for me!"
 	done
 
-UnknownText_0x1b522b:
+UnknownUnnecessaryText:
```

Comment:
The text is not unnecessary; the `farsjump` is unnecessary. `UnknownTougherThanEverText`.